src/main/java/org/opentripplanner/middleware/triptracker/TripStatus.java

```diff
@@ -96,33 +113,47 @@ public static double getSegmentStartTime(LegSegment legSegmentFromPosition) {
         return legSegmentFromPosition.cumulativeTime - legSegmentFromPosition.timeInSegment;
     }
 
+    /**
+     * Checks if the traveler's position is with an acceptable deviated distance of the mode type.
+     */
+    private static boolean isWithinModeDeviatedRadius(TravelerPosition travelerPosition) {
```

Comment:
See the other comment. I don't think we need that.

Reply:
Removed.